recipes/osgearth/all/conandata.yml

```diff
@@ -1,11 +1,17 @@
 sources:
+  '3.7.2':
+    sha256: 8bf241fa76e98ef3e072edb5da21ca213f7ef6be448905233e17eded60ccd71b
+    url: https://github.com/gwaldron/osgearth/archive/refs/tags/osgearth-3.7.2.tar.gz
   '3.3':
     sha256: 4b4a8ba70e707c6aae7d2fe2904b8761e9827398ddeb60633938fe486f5fa622
     url: https://github.com/gwaldron/osgearth/archive/refs/tags/osgearth-3.3.tar.gz
   '3.2':
     sha256: 7e1dd643b1f3b8d1ba9561b899c18176af988342b86c42d89a70be924cb747f6
     url: https://github.com/gwaldron/osgearth/archive/refs/tags/osgearth-3.2.tar.gz
 patches:
+  '3.7.2':
+    - patch_file: patches/3.7.2-osgearth.patch
```

Comment:
The [`conandata.yml` documentation](https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conandata_yml_format.md#patches-fields) would indicate that there are numerous problems with this new patch:

* There is no `patch_description` for this patch.
* There are no `patch_type` or `patch_source` fields for this patch. _(NOTE: These are only listed as recommended requirements.)_

I left this patch undocumented because it was created by migrating the changes from the other patches to v3.7.2, and said patches were themselves undocumented. Thus, I couldn't really tell you why (or even if, honestly) this patch needs to exist without further investigation.

That begs the question, though: Why aren't the other patches documented?

Reply:
The previous patch files were removed in commit 741efa258355c4cf8c971fe861076572a6818f6b. However, the `3.7.2-osgearth.patch` file remains undocumented for the reasons given above.